tests/js/browser-binder-tests.js

```diff
@@ -0,0 +1,639 @@
+// Test "binder" components using `gpii-test-browser`.
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-test-browser");
+gpii.tests.browser.loadTestingSupport();
+
+var path    = require("path");
+var url     = require("url");
+var fileUrl = url.resolve("file://", path.resolve(__dirname, "../static/tests-binder.html"));
+
+fluid.registerNamespace("gpii.binder.tests");
+
+// TODO:  If we find ourselves making model changes often, move this to `gpii-test-browser` and polish it up for wider use.
+gpii.binder.tests.applyChange = function (componentPath, modelPath, newValue) {
+    var component = fluid.getGlobalValue(componentPath);
+    component.applier.change(modelPath, newValue);
+};
+
+
+fluid.defaults("gpii.binder.tests.caseHolder", {
```

Comment:
To be clear, something like the WebDriver API offers us the ability to test things we cannot do with the current level of simulation in either (j)QUnit or Nightmare, namely actually testing tab navigation.  We can currently notify individual components that they are receiving specific keys, but we cannot execute a series of tab presses at the browser level and confirm that we pass through the expected areas of focus and end up in the expected area of focus.

These kinds of tests will always need to sit outside of QUnit, which as far as I know cannot be made to control a browser.

I would propose that we consider tab navigation as another "niche" and test it specifically with (for example) a package that wraps intern into a Fluid component that can be used with Fluid IoC tests.